src/apps/backend/modules/account/rest-api/account-router.ts

```diff
@@ -10,6 +10,8 @@ export default class AccountRouter extends ApplicationRouter {
 
     router.post('/', ctrl.createAccount);
 
+    router.patch('/:accountId/reset-password', ctrl.resetPassword);
```

Comment:
This is not a CRUD.

Can we just keep it PATCH? We allow updating password when user is not authenticated